src/services/item/plugins/shortLink/repository.ts

```diff
@@ -54,11 +54,20 @@ export const ShortLinkRepository = AppDataSource.getRepository(ShortLink).extend
     return shortLinks;
   },
 
-  async get(alias: string): Promise<ShortLink> {
+  async get(alias: string, returnItemIdOnly: boolean = false): Promise<ShortLink> {
     if (!alias) throw new ShortLinkNotFound(alias);
 
+    const selectItemIdOnly = {
+      select: {
+        item: {
+          id: true,
+        },
+      },
+    };
```

Comment:
I wonder if it would not be better to go as described here? https://github.com/typeorm/typeorm/issues/379 Otherwise you do a join without using it at all. Let me know if it changes to much stuff in your previous code.